### ISSUE TITLE
Application wasn't able to show games on the game list

### ISSUE THREAD

**Issue by dandure9** [DavidoTek/ProtonUp-Qt#436]
**Describe the bug**  
This is my first time using ProtonUp-Qt (2.9.2). I installed it via Flatpak, but the app does not display the game list when I press the button. I tried reinstalling it using the "command flatpak uninstall --delete-data <appid>", but the issue persists.

**To Reproduce**  
1. Open the app.
2. Click on 'Show Games List.'
3. The game list does not appear.
4. Click on 'Refresh Games.'
5. Nothing happens.

**Expected behavior**  
Loads games list for proton selection.

**Screenshots**  
N/A

**Desktop (please complete the following information):**  
 - **Platform**: PC/Laptop
 - **System**: Linux Mint virginia 21.3 x86_64
 - **Version**: ProtonUp-Qt 2.9.2
 - **How did you install ProtonUp-Qt?**: Flatpak 

**Additional context**  
The issue persists even after installing via the AppImage

**Terminal output**  
```
ProtonUp-Qt 2.9.2 by DavidoTek. Build Info: DavidoTek Flathub build.
Python 3.11.9 (main, Nov 10 2011, 15:00:00) [GCC 13.2.0], PySide 6.7.0
Platform: KDE Flatpak runtime 6.7 Linux-5.15.0-117-generic-x86_64-with-glibc2.38
Gtk-Message: 21:05:12.188: Failed to load module "xapp-gtk3-module"
Qt: Session management error: Could not open network socket
Loading locale en / en_US
Loaded ctmod GE-Proton
Loaded ctmod Wine-GE
Loaded ctmod Boxtron
Loaded ctmod D8VK (nightly)
Loaded ctmod Kron4ek Wine-Builds Vanilla
Loaded ctmod Lutris-Wine
Loaded ctmod Luxtorpeda
Loaded ctmod Northstar Proton (Titanfall 2)
Loaded ctmod Proton Tkg
Loaded ctmod Proton Tkg (Wine Master)
Loaded ctmod Roberta
Loaded ctmod Steam-Play-None
Loaded ctmod SteamTinkerLaunch
Loaded ctmod SteamTinkerLaunch-git
Loaded ctmod vkd3d-lutris
Loaded ctmod vkd3d-proton
Loaded ctmod Wine Tkg (Valve Wine Bleeding Edge)
Loaded ctmod Wine Tkg (Wine Master)
Loaded ctmod DXVK
Loaded ctmod DXVK Async
Loaded ctmod DXVK (nightly)
Error getting ctool map from appinfo.vdf: Invalid magic, got b')DV\x07'
Error updating SteamApp info from appinfo.vdf: Invalid magic, got b')DV\x07'
Gamepad error: No gamepad found.

```

**Comment by dandure9** [comment]
I know this is somewhat similar to  #155 , but I reviewed it and did not find a solution for my issue.

**Comment by sonic2kk** [comment]
This is because v2.9.2 is incompatible with the Steam Client Beta's new `appinfo.vdf` format. This is fixed in v2.10.0 which is not yet available as a Flatpak.

**Comment by dandure9** [comment]
Thank you so much, it works now!
